p2p/src/test/kotlin/maru/p2p/P2PTest.kt

```diff
@@ -66,33 +71,43 @@ import maru.p2p.ext.DataGenerators as P2P2DataGenerators
 class P2PTest {
   private val log: Logger = LogManager.getLogger(this.javaClass)
 
+  private var port1: UInt = 0u
+  private var port2: UInt = 0u
+  private var port3: UInt = 0u
+  private var port4: UInt = 0u
+  private var port5: UInt = 0u
+  private var port6: UInt = 0u
```

Comment:
I think we don't really need ports 1-3, because we can just use 0 when we create a P2P network and get an actual port from the instance itself